test/integration/src/py/test_ebrelayer_replay.py

```diff
@@ -78,11 +78,3 @@ def test_transfer_eth_to_ceth_using_replay_blocks(
     ending_balance2 = test_utilities.get_sifchain_addr_balance(request.sifchain_address, request.sifnodecli_node,
                                                                request.sifchain_symbol)
     assert (ending_balance2 == request.amount)
-
```

Comment:
Since ebrelayer doesn't record the block number in relayerdb, we shouldn't be doing this last part of the test.

Reply:
👍